README.org

```diff
@@ -62,21 +62,28 @@ If you use ELPA or MELPA, the easiest way to install =orderless= is via
 #+begin_src emacs-lisp
   (use-package orderless
     :ensure t
-    :custom (completion-styles '(orderless basic)))
+    :custom
+    (completion-styles '(orderless basic))
+    (completion-category-overrides '((file (styles basic partial-completion)))))
 #+end_src
 
 Alternatively, put =orderless.el= somewhere on your =load-path=, and use
 the following configuration:
 
 #+begin_src emacs-lisp
-(require 'orderless)
-(setq completion-styles '(orderless basic))
+  (require 'orderless)
+  (setq completion-styles '(orderless basic)
+        completion-category-overrides '((file (styles basic partial-completion))))
```

Comment:
I'd prefer orderless over partial-completion (I often forget the order of bits of long files names, "did I put my student's name first or the date first?") or maybe both orderless and partial-completion (for the compressed `/u/s/d/words` path handling).

Reply:
Oh, nevermind! I forgot "overrides" really means "additional"!

Reply:
Yes, that would work too. I just checked my config and I have `partial-completion` first, maybe such that partial paths and wildcards work more reliably. It is not a significant issue imo, but I've also discussed about this before with the Doom Emacs guys when they added Orderless to their configuration. I don't feel strongly about this, we can use one or the other.

Reply:
Yes it is really only a matter of partial-completion>orderless or orderless>partial-completion.